pull.py

```diff
@@ -31,18 +37,38 @@ def main():
         end_date = date.today()
         end_date = end_date.strftime('%Y-%m-%d')
 
-    stocks = pd.DataFrame()
+    ohlc = {}
+    mulstock = {}
+    tick = []
     for ticker in index:
         ts = TimeSeries(key='603MDIJGG9TGNV60', output_format='pandas')
         data, metadata = ts.get_daily(symbol='{}'.format(ticker))
         df = data
-        df = df.drop(['2. high', '3. low', '4. close', '5. volume'], axis=1)
-        df = df.rename(columns={'1. open': ticker})
+        df = df.drop(['5. volume'], axis=1)
+        df = df.rename(
+            columns={'1. open': 'Open', '2. high': 'High', '3. low': 'Low', '4. close': 'Close'})
+        ohlc[ticker] = go.Ohlc(
+            x=df.index,
+            open=df.Open,
+            high=df.High,
+            low=df.Low,
+            close=df.Close,
+            name=ticker)
+        df = df.transpose()
+        df['Stocks'] = ticker
+        df.index.name = 'Price'
+        df = df.set_index(['Stocks', df.index])
+        df = df.transpose()
+        df.index.name = 'Date'
+        mulstock[ticker] = df
+        tick.append(ticker)
+
+    stocks = pd.DataFrame()
+    for k in mulstock:
         if stocks.empty:
-            stocks = df
+            stocks = mulstock[k]
```

Comment:
You can collapse this down to one line:

stocks = mulstock[k] if stocks.empty else stocks.join(mulstock[k])